### PR TITLE
Add Sentry for Rails and React

### PR DESCRIPTION
**Pull request by wilsonw926**
Add Sentry for Rails and React

Adding pictures of a Destroy CaseNote Error. I've included additonal details like CaseNote attributes, Participant/Staff information, and Current user information

Error:
Screen Shot 2020-04-08 at 10 57 00 AM

User:
Screen Shot 2020-04-08 at 10 56 48 AM

CaseNote, Participant, Staff info:
Screen Shot 2020-04-08 at 10 56 37 AM

CC: @didvi @FranCogMents

### CODE REVIEW

**Comment by wilsonw926** [comment]
@JustinMi can i get an extra set of 👀 for the PR? Thanks!

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -3,13 +3,19 @@ class ApplicationController < ActionController::Base
...
+  before_action :set_raven_context
   after_action :verify_authorized, except: :index, unless: :devise_controller?
   after_action :verify_policy_scoped, only: :index
 
   rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
 
   private
 
+  def set_raven_context
+    Raven.user_context(id: session[:current_user_id])
```
[the remark this answers is not in the document]
Comment (wilsonw926) [reply app/controllers/application_controller.rb:15]: @didvi any thoughts on what else we should add?

**Comment by FranCogMents** [comment]
> 3. Let's add 1 [capture_exception](https://docs.sentry.io/error-reporting/capturing/?platform=browser) a component on the client side (maybe in a try/catch block or where error states are being logged to console)
> 
@wilsonw926 Actually -- after viewing Justin's PR -- instead of adding 1 capture_exception on the front-end let's create a logging function in the `axios.js` file and make a wrapper function for the captureExeption sentry function and apply it to at least one react component 🙌🏼

**Comment by wilsonw926** [comment]
@FranCogMents addressed all three things!

**Comment by wilsonw926** [comment]
Update to description: added pictures of Sentry dashboard for additional context

File: app/controllers/application_controller.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (didvi) [reply app/controllers/application_controller.rb]: Email works the best. If you still want my thoughts

**Pull request merged**